sources/updater/updog/src/main.rs

```diff
@@ -968,28 +927,49 @@ mod tests {
         let config = Config {
             metadata_base_url: String::from("foo"),
             targets_base_url: String::from("bar"),
-            seed: 512,
+            seed: Default::default(),
             version_lock: "latest".to_string(),
             ignore_waves: false,
         };
 
-        // Two waves; the 0th wave, and the final wave which starts in one hour
+        // Two waves; the 1st wave that starts immediately, and the final wave which starts in one hour
+        let time = Utc::now();
+        update.waves.insert(0, time);
         update
             .waves
-            .insert(1024, Utc::now() + TestDuration::hours(1));
+            .insert(1024, time + TestDuration::hours(1));
+        update
+            .waves
+            .insert(2048, time + TestDuration::hours(1));
         manifest.updates.push(update);
 
-        let potential_update =
-            update_required(&config, &manifest, &current_version, &variant, None)
-                .unwrap()
-                .unwrap();
-
         assert!(
-            potential_update.update_ready(512),
-            "0th wave doesn't appear ready"
+            update_required(
+                &manifest,
+                &current_version,
+                &variant,
+                config.ignore_waves,
+                0,
```

Comment:
This 0 is intended to match the seed from above, which we changed to `Default::default()`?  Is the value important?  If not, can we use `Default::default()` here too?  If it is, can we use a variable or something?